source/types/itemList.ts

```diff
@@ -38,3 +38,11 @@ export class ItemList<TItemType extends IItem> {
 		return this.items;
 	}
 }
+
+export class SortedItemList<TItemType extends IItem> extends ItemList<TItemType> {
+	setItems(items: TItemType[]): void {
+		super.setItems(items.sort((i1, i2) => {
+			return i1.display.localeCompare(i2.display);
```

Comment:
How about https://lodash.com/docs#sortBy? Might be simpler in the end. Something like
```
// top of file
import { sortBy } from 'lodash';

// setItems
sortBy(items, 'display');
or
sortBy(items, x => x.display);
```